cmd/createcluster.go

```diff
@@ -228,50 +223,40 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 	return nil
 }
 
-func createDepositData(secrets []*bls_sig.SecretKey, addr string, network string, numDVs int) ([]eth2p0.BLSPubKey, []eth2p0.BLSSignature, error) {
-	// TODO(xenowits): add flag to specify the number of distributed validators in a cluster
-	// Currently, we assume that we create a cluster of ONLY 1 Distributed Validator
-	var pubkeys []eth2p0.BLSPubKey
-	var msgSigs []eth2p0.BLSSignature
+// signDepositDatas returns a map of deposit data signatures by DV pubkey.
+func signDepositDatas(secrets []*bls_sig.SecretKey, withdrawalAddr string, network string) (map[eth2p0.BLSPubKey]eth2p0.BLSSignature, error) {
```

Comment:
instead of two slices that are implicitly linked, explicitly link pubkeys and signatures in a single map

Reply:
noted! is this the preferred approach?